Detectors/GlobalTrackingWorkflow/src/TOFEventTimeChecker.cxx

```diff
@@ -143,13 +204,20 @@ void TOFEventTimeChecker::fillMatching(GID gid)
   trk.expSigma[1] = 120;            // dummy resolution (to be updated)
   trk.expSigma[2] = 120;            // dummy resolution (to be updated)
 
-  //  int tofcl = match.getIdxTOFCl();
+  int tofcl = match.getIdxTOFCl();
   //  trk.mSignal = mTOFClustersArrayInp[tofcl].getTime();
   trk.mSignal = match.getSignal();
```

Comment:
troncamento da double a float

double tofsignal = match.getSignal();
static int nBC = 0;
if (mMyTracks.size() == 0){ // first track of the event (first in time) -> update nBC
  nBC = int(tofsignal * o2::tof::Geo::BC_TIME_INPS_INV);
}

trk.mSignal = float(tofsignal - double(o2::tof::Geo::BC_TIME_INPS)*nBC);

dovrebbe funzionare. Prova e fammi sapere